src/core/lib/iomgr/ev_epoll1_linux.cc

```diff
@@ -1227,6 +1279,18 @@ static const grpc_event_engine_vtable vtable = {
     shutdown_engine,
 };
 
+static void reset_event_manager_on_fork() {
```

Comment:
Some comments above this function would be nice. For example, a comment that this is called on child process after fork()

Reply:
Done